### ISSUE TITLE
A11y_Azure Tools Storage Explorer_ Toolbar_ Toolbar controls_Keyboard: Tooltip is not accessible by keyboard tab keys for "Toolbar" but when we hoverover using mouse it’s accessible

### ISSUE THREAD

**Issue by Jtith** [microsoft/AzureStorageExplorer#6370]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHub Tags:**
#A11yMAS; #A11yTCS; #MAC; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Mac_Dec2022; #Azure Tools Storage Explorer; #WCAG2.1.1; #Keyboard;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.27.0

### OS Details
MAC details: macOS Ventura 13.1

### Repro Steps:

1. Launch Storage Explorer
2. Tab to "Open Connect dialog" pane and Sign into Azure.
3. Tab to "Manage Accounts" pane and select the subscriptions you will use. Once selected, click on the “Apply" button.
4. Tab to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Tab to blob container "Toolbar" section 
6. and you will observe whether the tooltips are provided by navigating through keyboard tab keys.

### Actual results:
Tooltip is not accessible by keyboard tab keys for "Toolbar" but when we hoverover using mouse it’s accessible.
### Expected results:
Tooltips should be visible and accessible on navigating using keyboard tab key.
### User Impact:
Keyboard users may not get the information of the Toolbar button which are having tooltips but not accessible.

### Attachments:   
[tooltip.zip](https://github.com/microsoft/AzureStorageExplorer/files/10236058/tooltip.zip)

**Comment by MRayermannMSFT** [comment]
We should take this to office hours to downgrade to a sev 3/close as duplicate of https://github.com/microsoft/AzureStorageExplorer/issues/4987

**Comment by Jtith** [comment]
Closing as duplicate. https://github.com/microsoft/AzureStorageExplorer/issues/4987
